indico/modules/events/registration/fields/simple.py

```diff
@@ -66,10 +66,24 @@ class TextAreaField(RegistrationFormFieldBase):
     wtf_field_class = wtforms.StringField
 
 
-class SingleChoiceField(RegistrationFormBillableField):
+class ChoiceBaseField(RegistrationFormBillableField):
```

Comment:
After rebasing, inherit from `RegistrationFormBillableItemsField` instead of RegistrationFormBillableField` (that way you don't get unneeded price/is_billable keys in data since we only need it on items for these field types)